python/commons/lab.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import os
+import numpy as np
+
+import datajoint as dj
+
+schema = dj.schema('common_lab', locals())
+
+@schema
+class Paths(dj.Lookup):
+    definition = ...
+
+    def getLocalPath(self, path, local_os=None):
```

Comment:
Also this method doesn't use `self`, which usually suggests that this is better off as a `staticmethod`. Place `@staticmethod` right above the function definition, and you can skip the `self` argument.